pkg/auth/wildcard/match.go

```diff
@@ -5,7 +5,7 @@
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ *     https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
This is a verbatim copy of the license (supplied in the copy in MinIO, I believe).  Please revert this change.